ui/src/App.tsx

```diff
@@ -120,6 +124,17 @@ export default function App() {
       setCurFlowName(latestWf.name ?? null);
     }
     validateOrSaveAllJsonFileMyWorkflows();
+
+    // Execute after 10s to avoid conflict with validateOrSaveAllJsonFileMyWorkflows
+    scanLocalFileTimer.current = setTimeout(async () => {
```

Comment:
I think we should make validateOrSaveAllJsonFileMyWorkflows() async and awaitable, so that means inside validateOrSaveAllJsonFileMyWorkflows(), we will wait for all operations to finish (delete/create files) 

不然每个用户的网络环境不一样很难预估等待多久这个validateOrSaveAllJsonFileMyWorkflows可以完全完成

Reply:
已修改